apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -609,6 +649,7 @@ export const SignedVerdictOverview: React.FC = () => {
                 )}
                 isPrimaryButtonDisabled={isCaseModificationInvalid()}
                 handlePrimaryButtonClick={() => handleDateModification()}
+                isPrimaryButtonLoading={isSendingNotification}
```

Comment:
Consider modifying the condition to include isUpdatingCase. Otherwise, the button will be enabled for a brief moment while the client waits for the update, before the notification is sent.